perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -61,6 +80,7 @@
 flags.DEFINE_integer('sysbench_report_interval', 2,
                      'The interval, in seconds, we ask sysbench to report '
                      'results.')
+flags.DEFINE_integer('storage_size', 3400, 'Storage size for SQL instance.')
```

Comment:
specify the units for storage size, e.g., in GB

Also, does the default has to be this large? It has cost consequence. Probably we should specify a smaller default for those who just want to try out the benchmarks. 100?